pom.xml

```diff
@@ -152,6 +152,8 @@
                                 <version>${errorprone.version}</version>
                             </path>
                         </annotationProcessorPaths>
+                    	<fork>true</fork>
```

Comment:
I'm not seeing any performance improvements locally to setting `<fork>true</fork>`.  When I run `mvn -U clean package` on the latest `main` code (as of today), here are my results:
* (With current `main` branch, i.e. fork=false) Total time:  02:25 min
* (With this PR, i.e. fork=true)  Total time:  02:57 min

So, with fork=true, I'm actually seeing a slightly _slower_ build.  This is also the expected behavior noted [in this StackOverflow answer](https://stackoverflow.com/a/41270415) which says "true means it will create (fork) a new JVM to run the compiler. This is a bit slower but the isolation is better."